tests/10-verify.spec.js

```diff
@@ -349,11 +349,78 @@ describe('verify API (presentations)', () => {
   });
 });
 
+describe('tests for multiple credentials', async () => {
+
+  const credentialsCount = [5, 25, 50, 100];
+
+  for(const count of credentialsCount) {
+    it('cause error when credentials are tampered', async function() {
+      this.timeout(10000);
+      const challenge = uuid();
+      const {presentation, suite: vcSuite, documentLoader} =
+        await _generatePresentation({challenge, credentialsCount: count});
+
+      // tampering with the first two credentials id
+      presentation.verifiableCredential[0].id = 'some fake id';
+      presentation.verifiableCredential[1].id = 'some other fake id';
+
+      const result = await vc.verify({
+        documentLoader,
+        presentation,
+        suite: vcSuite,
+        unsignedPresentation: true
+      });
+
+      const credentialOne = result.credentialResults[0];
+      const credentialTwo = result.credentialResults[1];
+      const credentialThree = result.credentialResults[2];
+      const firstErrorMsg = result.credentialResults[0].error.errors[0].message;
+
+      result.verified.should.be.a('boolean');
+      result.verified.should.be.false;
+      credentialOne.verified.should.be.a('boolean');
+      credentialOne.verified.should.be.false;
+      credentialTwo.verified.should.be.a('boolean');
+      credentialTwo.verified.should.be.false;
+      credentialThree.verified.should.be.a('boolean');
+      credentialThree.verified.should.be.true;
+      firstErrorMsg.should.contain('Invalid signature.');
+    });
+
+    it('should not cause error when credentials are correct', async function() {
+      this.timeout(10000);
+      const challenge = uuid();
+      const {presentation, suite: vcSuite, documentLoader} =
+        await _generatePresentation({challenge, credentialsCount: count});
+
+      const result = await vc.verify({
+        documentLoader,
+        presentation,
+        suite: vcSuite,
+        unsignedPresentation: true
+      });
+
+      const credentialOne = result.credentialResults[0];
+      const credentialTwo = result.credentialResults[1];
+      const credentialThree = result.credentialResults[2];
+
+      result.verified.should.be.a('boolean');
+      result.verified.should.be.true;
+      credentialOne.verified.should.be.a('boolean');
+      credentialOne.verified.should.be.true;
```

Comment:
In this case, we expect that all the results should be true.  So please iterate over all the `credentialResults` and ensure that that verified is true in every case.